lib/view/page/profile_page.dart

```diff
@@ -0,0 +1,96 @@
+import 'package:flutter/material.dart';
+
+class ProfilePage extends StatefulWidget {
+  @override
+  _ProfilePageState createState() => _ProfilePageState();
+}
+
+class _ProfilePageState extends State<ProfilePage> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text('Profile'),
+        actions: [
+          PopupMenuButton(
+            itemBuilder: (context) => [
+              PopupMenuItem(
+                child: Text("Change Password"),
+                value: 'Change Password',
+              ),
+              PopupMenuItem(
+                child: Text("Log Out"),
+                value: 'Log Out',
+              )
+            ],
+            onSelected: (value) {
+              switch (value) {
+                case "Change Password":
+                  {
+                    print('change password');
+                  }
+                  break;
+                case "Log Out":
+                  {
+                    return showDialog(
+                      context: context,
+                      builder: (BuildContext context) {
+                        return AlertDialog(
```

Comment:
Logout dialog should be made into its own file in the /dialog directory. In case we add the ability to log out in multiple places.